src/View/ImageFusion/ImageFusionAxialView.py

```diff
@@ -260,3 +243,4 @@ def roi_display(self):
             polygons = self.patient_dict_container.get("dict_polygons_axial")[
                 roi_name][curr_slice]
             super().draw_roi_polygons(roi, polygons)
+            
```

Comment:
No newline at end of file

Reply:
I believe it would be fine just to leave a newline at the end of the file - also PEP8 recommends this, apparently. I just formatted the functionality using pep8 tool on pycharm and does not delete this newline. Also, upon investigating other files such as DicomAxialView.py, there is a newline at the end of the file.